images/saige-qtl/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM australia-southeast1-docker.pkg.dev/analysis-runner/images/driver-r:1.4
+
+ENV MAMBA_ROOT_PREFIX /root/micromamba
+
+SHELL ["/bin/bash", "-eo", "pipefail", "-c"]
+RUN wget -qO- https://api.anaconda.org/download/conda-forge/micromamba/0.8.2/linux-64/micromamba-0.8.2-he9b6cbd_0.tar.bz2 | tar -xvj -C /usr/local bin/micromamba && \
+    mkdir $MAMBA_ROOT_PREFIX && \
```

Comment:
You don't need to install `micromamba`, as you're already using the R driver image as a base.

Reply:
yes i see that now, thank you!